sci-libs/netcdf/netcdf47-4.7.2.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="Scientific library and interface for array oriented data access"
```

Comment:
Why not name this recipe `netcdf4.7`?

Reply:
I don't think you can have a dot in a filename (filename should be the same as in PROVIDES also).
Same as the ones for eg python, lua ...

Reply:
You can, and we should here. I think PROVIDES can have dots as well.

Reply:
Stand corrected, works fine by renaming it to netcdf4.7 (tested on 32bit nightly)